GDJS/Runtime/pixi-renderers/layer-pixi-renderer.ts

```diff
@@ -219,7 +217,7 @@ namespace gdjs {
         false
       );
       this._pixiRenderer.renderTexture.bind(
-        oldRenderTexture,
+        oldRenderTexture || undefined,
```

Comment:
this feels like a hack 🤔 
why is this needed?

Reply:
`renderTexture.bind()` only accepts `RenderTexture || undefined`, but `oldRenderTexture` is `RenderTexture || null`. This is needed to convert `oldRenderTexture`'s type signature to the one accepted by `renderTexture.bind()`. The null type of `oldRenderTexture` probably shouldn't be replaced by `undefined`, as it represent an empty value (`null`) and not an undefined value. Alternatively we could slap a ts-error-expected on that or modify the type definitions of PIXI to make it accept null as input value, but that feels like a bigger hack. We also cannot be 100% sure that PIXI checks for undefined (`if(oldRenderTexture=== undefined)`) or for truthiness (`if(oldRenderTexture)`), so it could be that passing in null instead of undefined could lead to something breaking (now or with a future PIXI update). Therefore I believe this is the beast solution to this.

Reply:
Sounds good then!